### PR TITLE
Correct grammar and clarify deployment sentence in documentation.

### PR DESCRIPTION
**Pull request by aswinayyolath**
Correct grammar and clarify deployment sentence in documentation.

### CODE REVIEW

File: src/content/getting-started/architecture/broker/_index.en.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ basically a set of Custom Resource Definitions (CRDs) backed by the Kubernetes d
 The Broker also defines a ServiceAccount and RBAC components to enable other Submariner
 components to securely access the Broker's API.
 
-While there no Services associated with the Broker, if using ```subctl``` to deploy the Broker, an
+While there are no services associated with the Broker, if you use ```subctl``` to deploy the Broker, an
```
Comment (vthapar) [inline src/content/getting-started/architecture/broker/_index.en.md]:
```suggestion
While there are no services associated with the Broker, using ```subctl``` to deploy the Broker
```

File: src/content/getting-started/architecture/broker/_index.en.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ basically a set of Custom Resource Definitions (CRDs) backed by the Kubernetes d
...
 operator Pod is also deployed that installs the CRDs and the Globalnet configuration.
```
Comment (vthapar) [inline src/content/getting-started/architecture/broker/_index.en.md]:
```suggestion
also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```

**Review by vthapar — CHANGES_REQUESTED** [review]
Avoid using 1st or 2nd person pronouns in documentation.

File: src/content/getting-started/architecture/broker/_index.en.md [hunk 3]
```diff
@@ -9,8 +9,8 @@ basically a set of Custom Resource Definitions (CRDs) backed by the Kubernetes d
 The Broker also defines a ServiceAccount and RBAC components to enable other Submariner
 components to securely access the Broker's API.
 
-While there no Services associated with the Broker, if using ```subctl``` to deploy the Broker, an
-operator Pod is also deployed that installs the CRDs and the Globalnet configuration.
+While there are no services associated with the Broker, using ```subctl``` to deploy the Broker
+also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```
Comment (skitt) [inline src/content/getting-started/architecture/broker/_index.en.md]:
```suggestion
While there are no Services associated with the Broker, using ```subctl``` to deploy the Broker
also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```

See [the Submariner docs style guide](https://submariner.io/development/website/style-guide/) and [the section on API objects in the Kubernetes docs style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).

**Review by skitt — CHANGES_REQUESTED** [review]

**Review by skitt — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Review by vthapar — APPROVED** [review]

**Pull request merged**